### PR TITLE
Update index.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**
For **editor's discretion** - Attempt to tersify the page, move background information down, and provide better/quicker overview on the tutorial content.

### CODE REVIEW

File: source/carousels/index.html.erb.md [hunk 1]
```diff
@@ -12,21 +12,21 @@ wcag_success_criteria:
...
 
-- **[Structure](structure.html):** Provide structural markup for the carousel and its items.
-
-- **[Functionality](functionality.html):** Add the functionality to display one carousel item at a time.
+- **[Functionality](functionality.html):** Add functionality to display one carousel item at a time.
 
 - **[Animations](animations.html):** Add animation and ensure users have control over these animations.
 
-- **[Full Content](examples/carousel.html):** The complete example source code.
+See also a [Complete Example](examples/carousel.html) including the full source code.
```
Comment (nitedog) [inline source/carousels/index.html.erb.md]: This is a different type of page, like the alt decision tree page in the images tutorial.

Comment (yatil) [inline source/carousels/index.html.erb.md]: Makes sense.

**Comment by yatil** [comment]
In general, I like the changes. However we now talk only little about the Carousel Concepts. I wonder if we can have a checklist-like list of considerations for carousels and link from there to detailed descriptions in the subpages.

I’ll try to come up with some changes and see if it works out.

**Comment by yatil** [comment]
Hi @nitedog, I have made some changes to tersify and reorder your proposal:

* Moved the definition of carousels up over the “Why is this important?” section to keep that section in sync with other such sections.
* Removed unnecessary phrases that I found to be more distracting than helpful. (Those have been there before Shadi’s changes.)

Find only my changes here: https://github.com/w3c/wai-tutorials/pull/361/commits/28a9b75d892a1a3a1cecd553e03a7ec2443a00cd

And the “rendered view”: https://github.com/w3c/wai-tutorials/blob/28a9b75d892a1a3a1cecd553e03a7ec2443a00cd/source/carousels/index.html.erb.md

**Pull request closed without merging**